eoscostarica/guide.eoscostarica.io
# Feat(frontend): GraphQL tutorial

### GH Issue

Resolves #223  

### Steps to test
1. Go to Tutorials/GraphQL
2. Check the content

#### CheckList
- [ ] The content is rigth
